src/modules/client/components/customers/infos/FundingDetailsModal.vue

```diff
@@ -30,9 +30,10 @@ export default {
   ],
   computed: {
     visibleColumns () {
-      const visibleColumns = this.selectedFunding.nature === FIXED
+      const visibleColumns = this.fundingDetailsData.nature === FIXED
```

Comment:
Pourquoi ne pas avoir simplement fait : `const visibleColumns = this.selected.nature === FIXED` ?

Reply:
D'ailleurs je renommerais `selected` en `funding`.

Reply:
Concernant le renommage, on définit aussi la props `selected` pour les modales d'historique de financement et d'historique de souscription. Je me dis que par soucis de clarté , ce serait peut être mieux de laisser `selected`, qu'en penses-tu?

Reply:
C'est renommé ;)